packages/server/src/comment/comment.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class CreateCommentInput {
+  @Field()
+  fileId: string;
+
+  @Field()
+  userId: string;
```

Comment:
We should get the user from the request instead of having it as an additional parameter